packages/medusa/src/api/routes/admin/orders/request-return.ts

```diff
@@ -264,3 +260,56 @@ export default async (req, res) => {
     throw err
   }
 }
+
+type ReturnObj = {
+  order_id?: string
+  idempotency_key?: string
+  items?: OrdersReturnItem[]
+  shipping_method?: ReturnShipping
+  refund_amount?: number
+  no_notification?: boolean
+}
```

Comment:
Question: Shouldn't the `ReturnObj`  be specified in the index file, in the same way that `AdminOrdersRes` is?

Reply:
It's not the object which is returned, it's an object used which represents a return